functions/index.js

```diff
@@ -77,3 +78,16 @@ exports.updateProfileCron = functions.https.onRequest((req, res) => {
       res.sendStatus(500);
     });
 });
+
```

Comment:
Maybe this should be written to the RTDB  here?   

in any case, would be good to have a comment here explaining what this is for (e.g. inputs to function:  config.client_id, config.client_secret and req.body.code and that the token returns is the users' github token which is needed by Firebase Auth to identify the user and also allows us to make calls on behalf of the user.